packages/design-system-docs/src/pages/components/Table/Table.example.jsx

```diff
@@ -0,0 +1,95 @@
+import {
+  Table,
+  TableBody,
+  TableCaption,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from '@cmsgov/design-system';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import uniqueId from 'lodash.uniqueid';
+
+const simpleHeaders = [
+  { title: 'Branch', key: 'branch', type: 'text', width: '20' },
+  { title: 'Article', key: 'article', type: 'text', width: '30' },
+  { title: 'Responsibility', key: 'responsibility', type: 'text', width: '50' },
+];
+
+const simpleData = [
+  {
+    branch: 'Legislative',
+    article: 'Article I of the Constitution establishes the legislative branch',
+    responsibility:
+      'Article I explains that Congress makes laws. Congress (the Senate and the House of Representatives) is the legislative branch of the U.S. government.',
+  },
+  {
+    branch: 'Executive',
+    article: 'Article II of the Constitution establishes the executive branch',
+    responsibility:
+      'The executive branch enforces the laws that Congress passes. The executive branch makes sure all the people follow the laws of the United States. The president is the head of the executive branch.',
+  },
+  {
+    branch: 'Judicial',
+    article: 'Article III of the Constitution establishes the judicial branch',
+    responsibility:
+      'One responsibility of the judicial branch is to decide if government laws and actions follow the Constitution.',
+  },
+];
+
+const tableId = uniqueId('tbl_');
+
+const renderHeader = () => {
+  return (
+    <TableRow>
+      {simpleHeaders.map((header, index) => {
+        return (
+          <TableHeader
+            key={header.key}
+            title={header.title}
+            type={header.type}
+            width={header.width}
+            scope="col"
+            id={tableId + '_col_' + index}
```

Comment:
I believe `uniqueId` should be called for each table header, something like this:
```
id={uniqueId('tbl_col_')}
```
Same goes for the uses of `tableId` below

Reply:
This `id` need to correspond to `header` - so need to just use one tableId - but just noticed that `uniqueId` may not work for us here for cases when we have multiple tables on the page - so will need to change to probably a random math unique id generation used for captionID

Reply:
`uniqueId` should always generate a unique id, regardless of if there are multiple tables. We already discussed this in our chat, but instead of using random ids here, we should just explicitly define clear readable values because this is an example.